src/pages/home/Form.jsx

```diff
@@ -32,13 +40,31 @@ const Signup = () => {
 				}}
 				validationSchema={SignUpSchema}
 				onSubmit={(values) => {
+					/* Functionality inspired by https://github.com/gndx/react-mailchimp-form */
+					const url =
+						'https://uwbacm.us16.list-manage.com/subscribe/post?u=8773dffc3f528b0b18fd7aef8&amp;id=7a3e5f69db';
+					const data = Object.keys(values)
+						.map((key) => `${key}=${encodeURIComponent(values[key])}`)
+						.join('&');
+					const path = `${url}&${data}`;
+					const endpoint = path.replace('/post?', '/post-json?');
+					console.log('posting to the following endpoint: ' + endpoint);
 					/* Post to the survey form */
-					fetch('https://uwbacm.us16.list-manage.com/subscribe/post?u=8773dffc3f528b0b18fd7aef8&id=11b31196c4', {
-						method: 'post',
-						body: values,
-						mode: 'no-cors'
-					}).then((response) => {
+					jsonp(endpoint, {param: 'c'}, (err, response) => {
+						if (response.msg.includes('already subscribed')) {
+							/* do something better here */
+							userMessage = submittedMessages.duplicate;
+						} else if (err) {
+							userMessage = submittedMessages.error;
+						} else if (response.result !== 'success') {
+							userMessage = submittedMessages.error;
+							console.log(response);
+						} else {
+							userMessage = submittedMessages.success;
+						}
 						hasSubmittedHandler(true);
+						// struggling to populate the response message in the form's div
+						console.log('user message is: ' + userMessage);
```

Comment:
I am unsure how to get the message value to populate without using an anti-pattern. @bpas247 any ideas?

Reply:
See the above comment, which I think is related to your issue.